File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/RowBasedCommand_ViewMyFlagsRow.cpp

#include "RowBasedCommand_ViewMyFlagsRow.h"
#include "RowBasedCommand_ViewMyFlagsRow.h"

namespace commands
{
  bool RowBasedCommand_ViewMyFlagsRow::getIsCheckboxesCheckBoxChecked( ) 
  {
    return this->isCheckboxesCheckBoxChecked;
  }
  int RowBasedCommand_ViewMyFlagsRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/RowBasedCommand_ViewMyFlagsRow.cs
/// <filename>
///     RowBasedCommand_ViewMyFlagsRow.cs
/// </filename>


namespace commands
{
    public class RowBasedCommand_ViewMyFlagsRow
  {
        private bool isCheckboxesCheckBoxChecked;
     
        public     virtual bool getIsCheckboxesCheckBoxChecked()
    {
      return this.isCheckboxesCheckBoxChecked;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
  }
}

